fast/stages/2-networking-b-vpn/variables.tf

```diff
@@ -199,3 +199,25 @@ variable "vpn_onprem_primary_config" {
   })
   default = null
 }
+
+variable "alert_config" {
+  description = "Configuration for monitoring alerts."
+  type = object({
+    vpn_tunnel_established = optional(object({
+      auto_close            = optional(string, null)
+      duration              = optional(string, "120s")
+      enabled               = optional(bool, true)
+      notification_channels = optional(list(string), [])
+      user_labels           = optional(map(string), {})
+    }), null)
```

Comment:
you don't need `null` as a default, it's implied :)

Reply:
Thanks, updated